data/netcdf_data.py

```diff
@@ -635,14 +642,19 @@ def dimensions(self) -> List[str]:
                     dimension_list = db.get_all_dimensions()
             except sqlite3.OperationalError:
                 pass
-            return dimension_list
-
+        
+        elif url.endswith(".zarr"):
```

Comment:
Again, what would happen here if `url` was indeed a list?

Reply:
If it's a list, it will be handled by line (new): 635